### PR TITLE
feat(NODE-4548): export ChangeStream class from top-level

### PR DESCRIPTION
**Pull request by nbbeeken**
#### What is changing?

NODE-4548
We need to make ChangeStream accessible

(Also cleaned up inconsistent naming in deprecated APIs)

#### What is the motivation for this change?

Backwards compat.

### Double check the following

- [x] Ran `npm run check:lint` script
- [x] Self-review completed using the [steps outlined here](https://github.com/mongodb/node-mongodb-native/blob/HEAD/CONTRIBUTING.md#reviewer-guidelines)
- [x] PR title follows the correct format: `<type>(NODE-xxxx)<!>: <description>`
- [x] Changes are covered by tests
- [x] New TODOs have a related JIRA ticket

### CODE REVIEW

**Review by dariakp — CHANGES_REQUESTED** [review]
Can you explain why the raw user options need to be exposed?

**Review by baileympearson — CHANGES_REQUESTED** [review]
Should this be a `feat` like Daria mentioned, and can we update the ticket description as well to reflect that you no longer are exporting the user options?

**Review by baileympearson — DISMISSED** [review]

**Review by dariakp — APPROVED** [review]
Confirmed the use in the legacy client, here: https://github.com/mongodb-js/nodejs-mongodb-legacy/pull/1 so this should be ready to go

File: src/collection.ts [hunk 1]
```diff
@@ -1610,13 +1610,13 @@ export class Collection<TSchema extends Document = Document> {
    * Updates documents.
    *
    * @deprecated use updateOne, updateMany or bulkWrite
-   * @param selector - The selector for the update operation.
+   * @param filter - The filter for the update operation.
```
Comment (durran) [inline src/collection.ts:1613]: Thanks for changing these to our spec language.

**Review by durran — APPROVED** [review]

**Pull request merged**